### ISSUE TITLE
No colors on istanbul report in console when using grunt task

### ISSUE THREAD

**Issue by stdavis** [theintern/intern#422]
intern 3.0.0-rc1

I'm not getting the extremely helpful colors in the istanbul report when I run my tests using the grunt task. I do see them when running intern directly.

Is this a limitation with grunt? If it's something that can be fixed I would be happy to give it a try if someone would point me in the right direction.

**Comment by csnover** [comment]
`process.stdout.isTTY` must be true for Istanbul to write coloured output, which it is not when spawning a Grunt process because it buffers output instead of passing through `process.stdout`. You could try adding `stdio: 'inherit'` to the options passed to `grunt.util.spawn` from the Intern task and see what happens. This could very easily break everything, however…

**Comment by stdavis** [comment]
@csnover You were right to be worried. Adding `stdio: 'inherit'` broke everything. :(

I did find out that Istanbul supports a `--color` command line flag that overrides the check on `process.stdout.isTTY` via [this pull request](https://github.com/gotwarlost/istanbul/pull/306). The only problem is that this was added to Istanbul at [v0.3.15](https://github.com/gotwarlost/istanbul/commit/7e2f37257a4cfdb5e75cbf6e0e19135637aef77c). Looks like Intern is currently using v0.2.16.

Is there any reason why intern is hanging back at v0.2.16 or am I free to submit a PR to upgrade it's dependency on Istanbul and add the `--color` flag to the grunt task. I was able to get the colored output working after making these changes.
